library/src/test/java/com/datatorrent/lib/io/fs/FileSplitterInputTest.java

```diff
@@ -499,7 +499,7 @@ public void testRecoveryOfBlockMetadataIterator() throws InterruptedException
     testMeta.fileSplitterInput.getScanner().setScanIntervalMillis(500);
 
 
-    testMeta.fileSplitterInput.setup(testMeta.context);
```

Comment:
It's not a point.  It's a question.  I am asking if similar changes are going to be made throughout the codebase?  
To me this change is completely  unnecessary. IMO code which was clean and intuitive is made the opposite just because a handful of unused objects will be present in stram.